dev-python/prompt_toolkit/prompt_toolkit-3.0.39.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="Library for building powerful interactive command line applications in Python"
+DESCRIPTION="prompt_toolkit could be a replacement for GNU readline, but it can be much \
+more than that.
+
+Some features:
+
+- Pure Python.
+- Syntax highlighting of the input while typing. (For instance, with a Pygments lexer.)
+- Multi-line input editing.
+- Advanced code completion.
+- Both Emacs and Vi key bindings. (Similar to readline.)
+- Even some advanced Vi functionality, like named registers and digraphs.
+- Reverse and forward incremental search.
+- Works well with Unicode double width characters. (Chinese input.)
+- Selecting text for copy/paste. (Both Emacs and Vi style.)
+- Support for bracketed paste.
+- Mouse support for cursor positioning and scrolling.
+- Auto suggestions. (Like fish shell.)
+- Multiple input buffers.
+- No global state.
+- Lightweight, the only dependencies are Pygments and wcwidth.
+- Runs on Linux, OS X, FreeBSD, OpenBSD and Windows systems.
+- And much more..."
+HOMEPAGE="https://github.com/prompt-toolkit/python-prompt-toolkit"
+COPYRIGHT="2014 Jonathan Slenders"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://pypi.io/packages/source/p/prompt_toolkit/prompt_toolkit-$portVersion.tar.gz"
+CHECKSUM_SHA256="04505ade687dc26dc4284b1ad19a83be2f2afe83e7a828ace0c72f3a1df72aac"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	$portName = $portVersion
```

Comment:
We keep ending up with empty base packages doing this, thought ... how about just using prompt_toolkit_python310 here instead of $portName and skip the eval sections?

Reply:
Alternative could be: https://bpa.st/OOKQ (link only stays up for a week)